lib/tasks/search_event_loader.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# Loaders can bulk load data
+namespace :search_events do
+  # csv loader can bulk load SearchEvents and Terms
+  #
+  # @note the csv should be formated as `term phrase`, `timestamp`. A dataclip is available that can export in this
+  #   format.
+  # @example
+  # bin/rails search_events:csv_loader['local_path_to_file.csv', 'some-source-to-use-for-all-loaded-records']
+  #
+  # @param path [String] local file path to a CSV file to load
+  # @param source [String] source name to load the data under
+  desc 'Load search_events from csv'
+  task :csv_loader, %i[path source] => :environment do |_task, args|
+    raise ArgumentError.new, 'Path is required' unless args.path.present?
+    raise ArgumentError.new, 'Source is required' unless args.source.present?
+
+    Rails.logger.info("Loading data from #{args.path}")
+
+    CSV.foreach(args.path) do |row|
+      term = Term.create_or_find_by!(phrase: row.first)
+      term.search_events.create!(source: args.source, created_at: row.last)
```

Comment:
You are correct that this would load the same csv multiple times if run multiple times and doesn't try to detect that it has already been loaded. I suspect this is fine, but maybe at minimum we should note?

Reply:
I just pushed a note to that effect. Will approve now so you can merge this as soon as you like, but feel free to modify the note as needed. (I'm not sure what multiple notes for the same method look like in YARD, and for some reason my local docserver isn't showing me the CSV loader task.)

Reply:
Yeah, the tasks don't show up in Yard sadly. I suspect we can nudge them in there with some config. Multiple notes are fine though from my poking around on other methods.